package.json

```diff
@@ -1,17 +1,8 @@
 {
   "private": true,
-  "publisher": "BroadcomMFD",
```

Comment:
Most likely because the root package.json is not part of the end product and its distribution. It is just a "glue" that hods the workspace together to be able to do one `yarn install`. The actual package.json for the extension is located in https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/master/packages/explorer-for-endevor/package.json#L7 .